commcare_connect/organization/models.py

```diff
@@ -43,6 +43,7 @@ class Role(models.TextChoices):
     role = models.CharField(max_length=20, choices=Role.choices, default=Role.MEMBER)
     invite_id = models.CharField(max_length=50, default=uuid4)
     accepted = models.BooleanField(default=False)
+    program_manager = models.BooleanField(default=False)
```

Comment:
Is this change a deviation from the Spec? The Spec mentions this to be on the Organization model instead and allowing all Admin Users in an Organization marked as Program Manager to have permissions to manage programs and create opportunities.